packages/@o3r/eslint-config/src/rules/typescript/unicorn.cjs

```diff
@@ -36,6 +36,7 @@ const config = [
       'unicorn/no-useless-promise-resolve-reject': 'off',
       'unicorn/prefer-set-has': 'off',
       'unicorn/prefer-string-slice': 'off',
+      'unicorn/prefer-top-level-await': 'off',
```

Comment:
Not working good with node < 22

Reply:
Do you mean <= or <?
![image](https://github.com/user-attachments/assets/ebc437ad-f4d9-4b65-8186-66689000995e)
Because today Node < 22 is no longer supported